### PR TITLE
docs(side-drawer): add an example with the header component

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: libs/components/src/lib/side-drawer/README.md [hunk 1]
```diff
@@ -294,6 +294,54 @@ To add content aside the side-drawer, add a slot called `app-content`.
 </script>
 ```
 
+## Example Usage
+
+### Side-drawer wrapping a [Header](../header)
```
Comment (rachelbt) [inline libs/components/src/lib/side-drawer/README.md:299]: I know this is the correct hierarchy but the text looks really big :(

**Review by rachelbt — COMMENTED** [review]
- the links in both readme doesn't work.
- There's something strange in the background of the code preview when scrolling - half is transparent and half has white bg.

**Comment by rinaok** [comment]
> * the links in both readme doesn't work.
> * There's something strange in the background of the code preview when scrolling - half is transparent and half has white bg:
>   image

@rachelbt The links work in localhost, so it's strange.

File: libs/components/src/lib/side-drawer/README.md (hunk 1 above)
Comment (yinonov) [reply libs/components/src/lib/side-drawer/README.md:299]: what do u suggest? keeping the text short? or resizing the font size?

**Comment by yinonov** [comment]
@AyalaBu u might wanna join in the review

File: apps/docs/assets/styles/main.css [hunk 2]
```diff
@@ -36,6 +36,7 @@ body a:visited {
 vwc-side-drawer {
   display: block;
   width: 100%;
+  height: 100%;
```
Comment (rinaok) [inline apps/docs/assets/styles/main.css:39]: @yinonov It fixed the mobile view issue. Send me a screenshot if you see any other issues.

**Review by yinonov — CHANGES_REQUESTED** [review]
part of the aside body in now hidden. notice the design system heading is not shown
image

**Comment by rinaok** [comment]
> part of the aside body in now hidden. notice the design system heading is not shown image

@yinonov changed it to be: calc(100% - 64px)

File: apps/docs/assets/styles/main.css (hunk 2 above)
Comment (rinaok) [reply apps/docs/assets/styles/main.css]: @yinonov changed it to: `height: calc(100% - 64px)`

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/header/README.md [hunk 3]
```diff
@@ -133,9 +133,9 @@ To add content below the header, add a slot called `app-content`.
 
 ## Example Usage
 
-### Fixed Header With Side-Drawer
+### Header wrapping a [Side-drawer](../../components/side-drawer)
```
[the remark this answers is not in the document]
Comment (rinaok) [reply libs/components/src/lib/header/README.md:136]: @yinonov  This is the title? `Header as Primary`

**Comment by yinonov** [comment]
think this should be closed considering #502?

**Pull request closed without merging**